docker-compose.yml

```diff
@@ -4,3 +4,5 @@ services:
     image: postgres:11.5-alpine
     ports:
       - '127.0.0.1:5433:5432'
+    volumes:
+      - ./bin/services/postgres:/opt/hypothesis
```

Comment:
I tried putting in the file, but it behaved weirdly. I think it might interpret things as a directory if they aren't already there. Not sure, this is actually kind of nicer anyway.